### PR TITLE
Wdp190705 17

### PR DESCRIPTION
**Pull request by EnDriash**

### CODE REVIEW

File: src/sass/sections/_newFurniture.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.section--products .row-products{
+    opacity: 0;
+}
```
Comment (stepek) [inline src/sass/sections/_newFurniture.scss]: Missing newline at the end of file

File: src/scripts/newFurniture.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+const product = require('./product_data.js');
+const productGenerator = require('./templatesGenerator.js')
+
+let tabs = document.querySelectorAll('.section--products .col.menu ul a');
```
Comment (stepek) [inline src/scripts/newFurniture.js]: Should be `const`

File: src/scripts/newFurniture.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    productFiller();
+    productGenerator(productToShow);
+});
+
+tabs.forEach((elem) => { // Added Event for every tab
+    elem.addEventListener('click', (event) => {
+        productToShow = [];
+        rowproducts.innerHTML = "";
+        fadeAnimation();
+        if(event.target !== active){ // Remove .active class for every tab
```
Comment (stepek) [inline src/scripts/newFurniture.js]: Please add newline before `if` block

File: src/scripts/newFurniture.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+            productFiller();
+            productGenerator(productToShow);
+        }
+    });
+});
+
+
+
+function fadeAnimation(){
```
Comment (stepek) [inline src/scripts/newFurniture.js]: Why don't you use CSS animation?

File: src/scripts/newFurniture.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        clearInterval();
+        } else{
+        opa += opa * 0.1;
+        rowproducts.style.opacity = opa;
+        }
+    }, 10);
+};
+
+function productFiller(){
+    for (var i = 0; i < product.length; i++) {
```
Comment (stepek) [inline src/scripts/newFurniture.js]: `let`

File: src/scripts/product_data.js [hunk 6]
```diff
@@ -1,8 +1,8 @@
 var product = [
```
Comment (stepek) [inline src/scripts/product_data.js]: const

**Comment by EnDriash** [comment]
How can i reverse changes for src/scripts/product_data.js and src/scripts/templatesGenerator.js when the changes was merged with master? I pull this refactoring for this task. I think it is impossible now.

File: src/scripts/product_data.js (hunk 6 above)
Comment (EnDriash) [reply src/scripts/product_data.js]: done

**Pull request merged**